e2e/geth-build/entrypoint.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env sh
```

Comment:
Does it make sense to use Ganache instead of geth so we don't have to use custom geth implementation?

Reply:
It is not custom geth implementation just regular private ethereum network. 
Ganache brings a lot of abstractions overhead on top of geth, while this solution is just clear private networks and you can easily control the version of network that you needs. 
Also it is works fine is there any way how ganache will make this work better?

Reply:
geth is just more clear for me, maybe there is a way to do this with Ganache, but i do not want to spend the time on it